actionpack/lib/abstract_controller/callbacks.rb

```diff
@@ -81,83 +81,107 @@ def _insert_callbacks(callbacks, block = nil)
       # :call-seq: before_action(names, block)
       #
       # Append a callback before actions. See _insert_callbacks for parameter details.
+      # Calling the same callback multiple times in +before_action+ will overwrite
+      # previous callback definitions.
 
       ##
       # :method: prepend_before_action
       #
       # :call-seq: prepend_before_action(names, block)
       #
       # Prepend a callback before actions. See _insert_callbacks for parameter details.
+      # Calling the same callback multiple times in +prepend_before_action+ will overwrite
+      # previous callback definitions.
```

Comment:
Not sure it needs to be on every *_action method - maybe just a note at the top?